### PR TITLE
Adds home and principles page copy

### PR DESCRIPTION
**Pull request by arnoldsandoval-okta**
- Updates homepage copy
- Adds principles page
- Adds Roadmap to sidebar

### CODE REVIEW

File: packages/docs/index.md [hunk 1]
```diff
@@ -19,32 +19,30 @@ contentPrimary:
     label: See them
     href: /icons/
 principle:
-  title: How we build
-  description: We welcome all feedback, designs, or ideas in order to produce the best possible experience for our users. If you’re interested in contributing, check out our contributing guidelines to get started.
-  label: Learn more
-  href: /base/
+  title: Purpose
+  description: Odyssey is Okta’s official design system built for use across all Okta products and sites. We aim to enable designers and developers to build efficiently and consistently while optimizing for user experience and accessibility.
```
Comment (jeffbelser-okta) [inline packages/docs/index.md:23]: _"To be bold and go where no Okta UI has gone before."_

 ✊🎤 ... ✋🎤

File: packages/docs/index.md [hunk 2]
```diff
@@ -19,32 +19,30 @@ contentPrimary:
...
-    href: /base/
-  - title: Our team
-    description: Et morbi eget at consectetur. Elit aenean mi phasellus. Et iusto odio atque dignissimos ducimus qui blanditiis praesentium voluptatum deleniti animi.
-    label: Learn more
-    href: /components/
-  - title: Get help
-    description: Et morbi eget at consectetur. Elit aenean mi phasellus. Et iusto odio atque dignissimos ducimus qui blanditiis praesentium voluptatum deleniti animi.
-    label: Learn more
-    href: /icons/
+    description: Our principles derive from Oktas design principles and guide us in making critical decisions as a team.
```
Comment (jeffbelser-okta) [inline packages/docs/index.md:37]:
```suggestion
    description: Our principles derive from Okta's design principles and guide us in making critical decisions as a team.
```

File: packages/docs/principles.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Okta is a technical product. It’s the role of Design to simplify unfamiliar and complex concepts. Ask questions, verify assumptions, dig into the needs and goals. Only once you understand the problem deeply can you solve it effectively.
+
+</Description>
+
+<Description>
+
+## Clear
+
+Our job is to eliminate ambiguity. We empower users to move forward with confidence, knowing what will happen next and why. Our designs should be clear over clever, digestible over exhaustive.
+
```
Comment (jeffbelser-okta) [inline packages/docs/principles.md:21]: well said here!

File: packages/vuepress-theme-odyssey/styles/components/_DocsSidebar.scss [hunk 4]
```diff
@@ -99,7 +99,7 @@ a.docs-site-title:visited {
 
   .docs-sidebar--footer {
     grid-area: footer;
-    padding: $spacing-xs $spacing-xs $spacing-m;
+    padding: $spacing-xs $spacing-xs 0;
```
Comment (jeffbelser-okta) [inline packages/vuepress-theme-odyssey/styles/components/_DocsSidebar.scss:102]: Consider submitting this work as a separate PRs to help clarity for reviewers!

**Review by jeffbelser-okta — APPROVED** [review]

**Review by edburyenegren-okta — APPROVED** [review]
👍  with @jeffbelser-okta's grammar edit!

**Pull request merged**